.travis/data/travis/.gitignore

```diff
@@ -0,0 +1,9 @@
+*
+
+# Don't ignore this file
+!.gitignore
+
+# Don't ignore data files
+!scurve
+!scurve/current
+!scurve/current/SCurveData.root
```

Comment:
Why is a duplicate `.gitignore` file needed at this location: `.travis/data/travis/`?

Reply:
I didn't want to add many long paths to the main `.gitignore`. I chose to move all data-related ignore directives into another file.

Reply:
I think this will be harder to maintain in the long run...now developers have to check 2 files instead of one.  And a new developer may not even know that a second `.gitignore` exists.  Let's revert this and use only one file.